magefile.go

```diff
@@ -88,6 +87,73 @@ func fetchPackageStorage() error {
 		packageStorageRevision)
 }
 
+func buildPackages() error {
```

Comment:
Looking forward for this one to also disappear!

Reply:
Yes, we have a plan for this one: https://github.com/elastic/package-registry/issues/505